composer.json

```diff
@@ -17,7 +17,7 @@
         "laravel/sail": "^1.41.0",
         "mockery/mockery": "^1.6.12",
         "nunomaduro/collision": "^8.6.1",
-        "phpunit/phpunit": "^11.5.6"
+        "phpunit/phpunit": "^11.5.7"
```

Comment:
Maybe should we use Pest instead?

Reply:
I used the `laravel new` command to generate this skeleton, as you can see on the official `laravel/laravel`  repo: https://github.com/laravel/laravel/blob/cf5aed8c2e7fe9ae038e76ccd0ac8de1cdc3dded/composer.json#L20